cider/tests/functionality/CiderFilterTest.cpp

```diff
@@ -398,117 +611,135 @@ TEST_F(CiderFilterSequenceTestArrow, ArrowMultiFilterWithAndTest) {
 }
 
 TEST_F(CiderFilterSequenceTestArrow, ArrowMultiColEqualTest) {
-  prepareArrowBatch();
   assertQueryArrow("SELECT col_1, col_2 FROM test WHERE col_1 = col_2");
   assertQueryArrow("SELECT col_2, col_3 FROM test WHERE col_2 = col_3");
   assertQueryArrow("SELECT col_2, col_4 FROM test WHERE col_2 = col_4");
   assertQueryArrow("SELECT col_3, col_4 FROM test WHERE col_3 = col_4");
 }
 
-TEST_F(CiderFilterRandomTestBase, multiColRandomTest) {
-  assertQuery("SELECT col_1, col_5 FROM test WHERE col_1 < col_5");
-  assertQuery("SELECT col_2, col_6 FROM test WHERE col_2 < col_6");
-  assertQuery("SELECT col_3, col_7 FROM test WHERE col_3 <= col_7");
-  assertQuery("SELECT col_4, col_8 FROM test WHERE col_4 <= col_8");
-  assertQuery("SELECT col_1, col_5 FROM test WHERE col_1 <> col_5");
-  assertQuery(
+TEST_F(CiderFilterRandomTestArrow, multiColRandomTest) {
```

Comment:
should we keep original test cases?

Reply:
yes, all original test cases are on top of this file.

Reply:
oh sorry. I didn't notice that. Can we re-org to reduce code diff?

Reply:
never mind, original test cases interspersing with the new ones will reduce code diff, but we have to go into each case to find and remove the original ones when finished arrow migration.